### PR TITLE
Verifier cleanup/bugfix crop workaround

### PR DESCRIPTION
**Pull request by Jakub89**
Resolves:
https://codepoets.atlassian.net/browse/GF-43
https://codepoets.atlassian.net/browse/GF-88

### CODE REVIEW

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/crop_generator.py [hunk 1]
```diff
@@ -82,38 +95,55 @@ def get_default_crop_size(self) -> Tuple[int, int]:
...
+    def __init__(
+            self,
+            id_: int,
+            subimage: SubImage,
+            pixel_region=None,
+            crop_region=None
+    ) -> None:
+        self.id = id_
+        self.subimage = subimage
+        self.pixel_region = pixel_region if pixel_region is not None else None
```
Comment (dybi) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/crop_generator.py]: if `None` is default value, the conditional part seems unnecessary; below as well

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/crop_generator.py [hunk 2]
```diff
@@ -171,3 +214,66 @@ def _get_random_interval_within_boundaries(begin: int,
...
     return interval_begin, interval_end
+
+
+def generate_single_random_crop_data(  # pylint: disable=too-many-locals
+        subimage: SubImage, id_: int
+) -> Crop:
+
+    crop_size_x = 0.1
+    crop_size_y = 0.1
+    # check resolution, make sure that cropp is greather then 8px.
```
Comment (dybi) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/crop_generator.py]: "cropp" -> "crop"

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py [hunk 3]
```diff
@@ -59,34 +67,50 @@ def calculate_metrics(reference_img_path,
         default_metrics['Label'] = VERIFICATION_FAIL
     if default_metrics['Label'] == VERIFICATION_SUCCESS:
```
Comment (dybi) [inline apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: this `if` now seems unnecessary. Regardless of of the verification's results, please save crop and return path to the saved metrics.

**Review by dybi — COMMENTED** [review]
Please remove `get_crops()` as it is not needed anymore. Instead use single crop.

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/crop_generator.py (hunk 1 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/crop_generator.py]: Done

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/crop_generator.py (hunk 2 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/crop_generator.py]: Done

File: apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py (hunk 3 above)
Comment (Jakub89) [reply apps/blender/resources/images/entrypoints/scripts/verifier_tools/img_metrics_calculator.py]: Done

**Review by dybi — APPROVED** [review]
girl_approves

**Pull request merged**